Sources/Views/Collections/CollectionIndex.swift

```diff
@@ -7,41 +7,184 @@ import PointFreeRouter
 import Styleguide
 import Prelude
 
-public let collectionIndex: Node = .gridRow(
-  attributes: [
-    .style(
-      maxWidth(.px(1080))
-        <> margin(leftRight: .auto)
+public func collectionIndex(
+  collections: [Episode.Collection]
+) -> Node {
+  [
+    .gridRow(
+      attributes: [
+        .class([
+          Class.padding([.mobile: [.top: 3], .desktop: [.top: 4]]),
+          Class.grid.between(.desktop)
+        ]),
+        .style(
+          maxWidth(.px(1080))
+            <> margin(leftRight: .auto)
+        )
+      ],
+      .gridColumn(
+        sizes: [.mobile: 12],
+        attributes: [
+          .class([
+            Class.padding([.mobile: [.bottom: 2, .leftRight: 2], .desktop: [.leftRight: 3]])
+          ])
+        ],
+        .h3(
+          attributes: [
+            .class([Class.pf.type.responsiveTitle2])
+          ],
+          "Collections"
+        )
+      )
+    ),
+    .ul(
+      attributes: [
+        .class([
+          Class.margin([.mobile: [.all: 0]]),
+          Class.padding([.mobile: [.all: 0], .desktop: [.leftRight: 2, .topBottom: 0]]),
+          Class.type.list.styleNone,
+          Class.flex.wrap,
+          Class.flex.flex
```

Comment:
the tricky part about the design of the collections index is that in each row if one item is bigger than the other they should both grow to be the same size. I did this for the [pricing](http://www.pointfree.co/pricing) page, and it took some time to get rid, and so I just copied it for this one.

with a bit of time I could clean this up, but let's just roll with it for now